distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1185,6 +1187,66 @@ public void testRetrievalReliablility() throws Exception {
     }
 
     /**
+     * Tests that if the endpoint disconnects twice, the retrieval retries both times
+     * This test will respond with the correct Partial Content when a range header is sent in the request
+     * @throws Exception
+     */
+    @Test
+    @Ignore
```

Comment:
We should add the DDF ticket number and explain why this test is ignored in the comments.